mariadb/values.yaml

```diff
@@ -1,6 +1,20 @@
-imageTag: 10.1.14-r1
+# Bitnami MariaDB image version
+# ref: https://hub.docker.com/r/bitnami/mariadb/tags/
+imageTag: 10.1.14-r2
 
+# Specify a imagePullPolicy
+## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+imagePullPolicy:
```

Comment:
Yep, either they should be commented, or they should be set to the default value.